spec/classes/init_spec.rb

```diff
@@ -41,8 +41,15 @@
               it { is_expected.to contain_concat_fragment('squid_header').with_content(%r{^access_log\s+daemon:/var/log/squid/access.log\s+squid$}) }
             end
           end
+        when 'FreeBSD'
```

Comment:
agreed.  apologies...i should have realized that.  do you want a new PR/branch for that?